<!DOCTYPE html>
<html lang="pt">
<head>
  <meta charset="UTF-8">
  <title>D. Pratas @ Interdisciplinary Informatics</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="css/styles.css">
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.5.2/css/all.min.css">
</head>
<body>

  <header>
    <h1>D. Pratas @ Interdisciplinary Informatics</h1>
  </header>

  <nav>
    <ul>
      <li><a href="#About">About</a></li>
      <li><a href="#Research">Research</a></li>
      <li><a href="#Students">Students</a></li>
      <li><a href="#Publications">Publications</a></li>
      <li><a href="#Software">Software</a></li>
      <li><a href="#Datasets">Datasets</a></li>
      <li><a href="#Classes">Classes</a></li>
      <li><a href="#Challenge">Challenge</a></li>
      <li><a href="#Contact">Contact</a></li>
    </ul>
  </nav>

  <main>
    <section id="About">
      <h2>About</h2>
        <p align="justify">
          Diogo Pratas is a researcher in interdisciplinary informatics, dedicated to developing computational methods that bridge informatics with biomedical, anthropological, and historical research. His work focuses on analyzing and interpreting complex data. Current projects include efficient compression of biological data, reconstruction and analysis of viral genomes, metagenomic studies of ancient DNA, and the application of machine learning to date and localize ancient artifacts such as metals, texts, and genetic material.
        </p>

        <div class="social-icons">
          <a href="https://www.researchgate.net/profile/Diogo-Pratas" target="_blank" title="ResearchGate">
          <i class="fab fa-researchgate fa-2x"></i>
          </a>
          <a href="https://scholar.google.com/citations?user=HasPwO0AAAAJ" target="_blank" title="Google Scholar">
          <i class="fas fa-graduation-cap fa-2x"></i>
          </a>
          <a href="https://orcid.org/0000-0003-1176-552X" target="_blank" title="ORCID">
          <i class="fab fa-orcid fa-2x"></i>
          </a>
          <a href="https://www.scopus.com/authid/detail.uri?authorId=49361962700" target="_blank" title="Scopus">
          <i class="fas fa-book fa-2x"></i>
          </a>
          <a href="https://www.linkedin.com/in/diogo-pratas" target="_blank" title="LinkedIn">
          <i class="fab fa-linkedin fa-2x"></i>
          </a>
          <a href="https://github.com/pratas" target="_blank" title="GitHub">
          <i class="fab fa-github fa-2x"></i>
          </a>
          <a href="https://twitter.com/dgpratas" target="_blank" title="X">
          <i class="fab fa-x-twitter fa-2x"></i>
          </a>
          <a href="https://bsky.app/profile/dgpratas.bsky.social" target="_blank" title="Bluesky">
          <i class="fas fa-cloud fa-2x"></i>
          </a>
          <br><br>
          <button id="toggleBiography">Show Biography</button>
          <ul id="biography" class="hidden">
            <p align="justify">
            Diogo Pratas was born in Aveiro, in 1986.
            He graduated in Information and Communication Technologies at the University of Aveiro, in 2008. During his degree, he participated in the Erasmus program in Computer Engineering at the Pontifical University of Salamanca, Spain. He worked in the private sector from 2008 to 2010 in the areas of ​​networks and IT security and in the development of Linux systems. In 2016, he obtained his PhD in Informatics at the University of Aveiro with a thesis on compression and analysis of genomic sequences. He did a post-doctorate in Computer Science from 2016 to 2019 at the University of Aveiro.
            In 2019, he was a Staff Bioinformatician at the University of Helsinki, Finland.
            Since the end of 2019, he has been an Investigador Auxiliar at the University of Aveiro in the area of Interdisciplinary Informatics. Since 2022, he has been a visiting researcher at the Department of Virology at the University of Helsinki, Finland. Since 2019, he teaches Algorithmic Information Theory at the Department of Electronics, Telecommunications, and Informatics at the University of Aveiro.
            He has organized several scientific conferences, workshops, and competitions, including the International Conference on Algorithms for Computational Biology, the Workshop on Genomics for Physicians, and the Portuguese League of Bioinformatics.
            He is member of several scientific associations, including the European Society for Clinical Virology (ESCV) and the Portuguese Association for Pattern Recognition (APRP).
            His main areas of interest and research are Bioinformatics, Computational virology, Cultural Heritage, Artificial Inteligence, and Information Theory. 
            He has participated as an invited speaker at various international conferences and scientific meetings and is the author of more than one hundred publications in the areas of  Medicine, Biology, and Informatics.
            </p>
          </ul>
        </div>
    </section>

    <section id="Research">
      <h2>Research</h2>
      <ul>
        <li><p align="justify"><strong>Bioinformatics:</strong> Advanced genetic and structural analysis through the development of efficient computational tools. This includes genome classification, identification of patterns in biological sequences, and the reconstruction of ancient and modern genomes (mostly viruses).</p></li>
        <li><p align="justify"><strong>Computational Biology:</strong> Modeling and simulating biological systems to explore functional genomics and evolutionary dynamics. This line involves developing interdisciplinary methods and tools to interpret complex biological patterns across species and time scales, and includes applied solutions to environmental challenges, such as plastic biodegradation.</p></li>
        <li><p align="justify"><strong>Computational Medicine:</strong> Development and application of innovative computational methods for virus detection in human tissues, virome analysis (including in disease and transplantation contexts), cytogenetic analysis, and identification of therapeutic targets, contributing to diagnosis, prognosis, and treatment in medicine.</p></li>
        <li><p align="justify"><strong>Artificial Intelligence for Cultural Heritage:</strong> Application of machine learning and data mining techniques to the preservation and digital analysis of traditional knowledge. Projects involve dating and localize ancient artifacts, such as metals, texts, and DNA.</p></li>
        <li><p align="justify"><strong>Data Compression and Analysis:</strong> Development of efficient algorithms to process and interpret large-scale datasets, emphasizing statistical and algorithmic methods, parameter optimization, and machine learning to enable data reduction, pattern discovery, and the extraction of meaningful structure from complex data.</p></li>
        <li><p align="justify"><strong>Algorithmic Information Theory:</strong> Development of methodologies for generating and analyzing pseudo-random Turing Machines to identify those that produce statistically complex tape behaviors, along with the development of computational approaches to discover short Turing Machines that describe exact or approximate digital objects and to compute approximations of their logical depth.</p></li>    
      </ul>
    </section>

    <section id="Students">
      <h2>Students</h2>
        <h3>PhD Student</h3>
          <ul>
            <li>M. J. P. Sousa - Intelligent reconstruction and analysis of viral genomes.</li>
          </ul>
        <h3>Graduated Student</h3>
          <ul>
            <li>D. Yamunaque - Estimating the Age and Provenance of Ancient DNA with Machine Learning.</li>
          </ul>
        <h3>Master Students</h3>
          <ul>
            <li>A. Gomes - Human Virus Genomics and distribution.</li>
            <li>B. Simões - Characterization and visualization of plant genomes using machine learning.</li>
            <li>D. Lourenço - Exploring microalgae-enzymes for sustainable plastic biodegradation.</li>
            <li>D. Silva - Detection of Machine-Generated Text.</li>
            <li>J. Gaspar - Compression of Astronomical Data.</li>
            <li>L. Marques - Automatic DNA classification of organisms contained in ancient samples.</li>
            <li>P. Pinto - Text dating using data compression and machine learning.</li>
            <li>R. Dias - Complexity analysis of musics.</li>
            <li>S. Almeida - Analysis of Relative Absent Words for Innovative Diagnosis.</li>
          </ul>
        <!--<h3> Undergraduate Student</h3>
          <ul>
            <li>R. Gomes - Human Virus Genomics and distribution.</li>
          </ul>-->
        <h3>Thesis and Dissertations</h3>
          <ul>
            <li>Denis Yamunaque - <a href="">A machine learning approach for authentication of ancient DNA samples</a> (2025).</li>
            <li>Ricardo Dias - <a href="">Machine Learning Approach for Non-Destructive Coin Dating</a> (2025).</li>
            <li>Rita Ferrolho - <a href="http://hdl.handle.net/10773/44662">Optimization of a genomic data compressor using metameric genetic algorithms</a> (2024).</li>
            <li>Clara Cerqueira - <a href="https://hdl.handle.net/10216/164920">Machine learning-enhanced optimization of plastic-degrading enzymes for sustainable ocean cleanup</a> (2024).</li>
            <li>Dinis Lei - <a href="http://hdl.handle.net/10773/42915">Study of the impact of data compression on energy consumption reduction</a> (2024).</li>
            <li>Margarida Pinheiro - <a href="https://hdl.handle.net/10216/164527">Genomic diversity and zoonotic potential of hepatitis E virus in European rabbits: implications for diagnostic and therapeutic approaches</a> (2024).</li>
            <li>Mariana Fernandes - <a href="https://hdl.handle.net/10216/165579">Designing optimal 3D enzyme computational models for efficient plastic degradation</a> (2024).</li>
            <li>Rafael Vieira - <a href="https://hdl.handle.net/10216/163263">Designing in-silico aptamers for potential use in marine bioremediation</a> (2024).</li>
            <li>Maria J. P. Sousa - <a href="http://hdl.handle.net/10773/40666">Automatic reconstruction of persistent human virus sequences</a> (2023).</li>
            <li>Renato Soares - <a href="https://hdl.handle.net/10216/152564">Improving a database of cyanobacterial bioactive compounds that can be used for therapeutic approaches in human diseases</a> (2023).</li>
            <li>Tiago Fonseca - <a href="http://hdl.handle.net/10773/41955">Impact of sorting in DNA sequence compression</a> (2023).</li>
            <li>Jorge M. Silva - <a href="http://hdl.handle.net/10773/41759">Algorithmic information approximations in data analysis</a> (2023).</li>
            <li>Alexandre Lourenço - <a href="http://hdl.handle.net/10773/33658">Reconstruction and classification of unknown DNA sequences</a> (2021).</li>
            <li>Milton Silva - <a href="http://hdl.handle.net/10773/30792">Efficient biosequence compression using neural networks</a> (2021).</li>
            <li>Morteza Hosseini - <a href="http://hdl.handle.net/10773/29916">Compression models and tools for omics data</a> (2020).</li>
            <li>Manuel Gaspar - <a href=" http://hdl.handle.net/10773/23810">Automatic system for approximate and noncontiguous DNA sequences search</a> (2017).</li>
          </ul>
    </section>
    
    <section id="Publications">
      <h2>Publications</h2>
      <h3>Journal Articles:</h3>
        <ol>
          <li>M. J. P. Sousa, A. J. Pinho, <strong>D. Pratas</strong>*. <a href="https://doi.org/10.1093/bioinformatics/btae725">JARVIS3: an efficient encoder for genomic data</a>. <strong>Bioinformatics, 2024</strong>.</li>
          <li>J. M. Silva, A. J. Pinho, <strong>D. Pratas</strong>*. <a href="https://doi.org/10.1093/gigascience/giae086">AltaiR: a C toolkit for alignment-free and temporal analysis of multi-FASTA data</a>. <strong>GigaScience, 2024</strong>.</li>
          <li>R. Soares, L. Azevedo, V. Vasconcelos, <strong>D. Pratas</strong>, S. Sousa, J. Carneiro. <a href="https://doi.org/10.1021/acs.jcim.4c00995">Machine Learning-Driven Discovery and Database of Cyanobacteria Bioactive Compounds: A Resource for Therapeutics and Bioremediation</a>. <strong>Journal of Chemical Information and Modeling, 2024</strong>.</li>
          <li>L. Pyöriä, <strong>D. Pratas</strong>, M. Toppinen, P. Simmonds, K. Hedman, A. Sajantila, M. F. Perdomo. <a href="https://doi.org/10.1093/nar/gkae871">Intra-host genomic diversity and integration landscape of human tissue-resident DNA virome</a>. <strong>Nucleic Acids Research, 2024</strong>.</li>
          <li>L. Hannolainen, L. Pyöriä, <strong>D. Pratas</strong>, J. Lohi, S. Skuja, S. Rasa-Dzelzkaleja, M. Murovska, K. Hedman, T. Jahnukainen, M. F. Perdomo. <a href="https://www.duodecimlehti.fi/xmedia/duo/duo18264.pdf">Perinnöllinen herpesvirus elinsiirron kiusana</a>. <strong>Duodecim, 2024</strong>.</li>
          <li>L. Hannolainen, L. Pyöriä, <strong>D. Pratas</strong>, J. Lohi, S. Skuja, S. Rasa-Dzelzkaleja, M. Murovska, K. Hedman, T. Jahnukainen, M. F. Perdomo. <a href="https://doi.org/10.1093/infdis/jiae268">Reactivation of a transplant recipient’s inherited human herpesvirus 6 and implications to the graft</a>. <strong>The Journal of Infections Diseases, 2024</strong>.</li>
          <li>J. M. Silva, W. Qi, A. J. Pinho, <strong>D. Pratas</strong>*. <a href="https://doi.org/10.1093/gigascience/giad101">AlcoR: alignment-free simulation, mapping, and visualization of low-complexity regions in biological data</a>. <strong>GigaScience, 2023</strong>.</li>
          <li>J. Carneiro, F. Pascoal, M. Semedo, <strong>D. Pratas</strong>, M. P. Tomasino, A. Rego, M. F. Carvalho, A. P. Mucha, C. Magalhães. <a href="https://doi.org/10.1016/j.envres.2023.116040">Mapping human pathogens in wastewater using a metatranscriptomic approach</a>. <strong>Environmental Research, 2023</strong>.</li>
          <li>J. Carneiro, R. P. Magalhães, V. M. de la Oliva Roque, M. Simões, <strong>D. Pratas</strong>, S. Sousa. <a href="https://doi.org/10.1007/s10822-023-00505-5">TargIDe: a machine-learning workflow for target identification of molecules with antibiofilm activity against Pseudomonas aeruginosa</a>. <strong>J Comput Aided Mol Des, 2023</strong>.</li>
          <li>L. Pyöriä, <strong>D. Pratas</strong>, M. Toppinen, K. Hedman, A. Sajantila, M. F. Perdomo. <a href="https://doi.org/10.1093/nar/gkad199">Unmasking the Tissue-Resident Eukaryotic DNA Virome in Humans</a>. <strong>Nucleic Acids Research, 2023</strong>.</li>
          <li>L. Pyöriä, <strong>D. Pratas</strong>, M. Toppinen, K. Hedman, A. Sajantila, M. F. Perdomo. <a href="https://helda.helsinki.fi/bitstreams/dd6f7c9b-1402-4289-acb3-ce1c60ecb4b7/download">Elimistömme on lukuisten terveyteemme vaikuttavien virusten koti</a>. <strong>Duodecim, 2023</strong>.</li>
          <li>M. K. Jauhiainen, U. Mohanraj, M. Lehecka, M. Niemelä, T. P. Hirvonen, <strong>D. Pratas</strong>, M. F. Perdomo, M. Söderlund-Venermo, A. A. Mäkitie, S. T. Sinkkonen. <a href="https://doi.org/10.1007/s13365-023-01112-8">Herpesviruses, polyomaviruses, parvoviruses, papillomaviruses, and anelloviruses in vestibular schwannoma</a>. <strong>Journal of NeuroVirology, 2023</strong>.</li>
          <li>J. M. Silva, <strong>D. Pratas</strong>, T. Caetano, S. Matos. <a href="https://doi.org/10.1093/gigascience/giac079">The complexity landscape of viral genomes</a>. <strong>GigaScience, 2022</strong>.</li>
          <li>W. Qi, Y. Lim, A. Patrignani, P. Schläpfer, A. Bratus-Neuenschwander, S. Grüter, C. Chanez, N. Rodde, E. Prat, S. Vautrin, M. Fustier, <strong>D. Pratas</strong>, R. Schlapbach, W. Gruissem. <a href="https://doi.org/10.1093/gigascience/giac028">The haplotype-resolved chromosome pairs of a heterozygous diploid African cassava cultivar reveal novel pan-genome and allele-specific transcriptome features</a>. <strong>GigaScience, 2022</strong>.</li>
          <li>O. I. Mielonen, <strong>D. Pratas</strong>, K. Hedman, A. Sajantila, M. F. Perdomo. <a href="https://doi.org/10.3390/v14010133">Detection of Low-Copy Human Virus DNA upon Prolonged Formalin Fixation</a>. <strong>Viruses, 2022</strong>.</li>
          <li>M. Toppinen, A. Sajantila, <strong>D. Pratas</strong>, K. Hedman, M. F. Perdomo. <a href="https://doi.org/10.3389/fcimb.2021.657245">The Human Bone Marrow Is Host to the DNAs of Several Viruses</a>. <strong>Frontiers in cellular and infection microbiology, 2021</strong>.</li>
          <li>J. Monteiro, <strong>D. Pratas</strong>, A. Videira, F. Pereira. <a href="https://doi.org/10.1111/lam.13538">Revisiting the Neurospora crassa mitochondrial genome</a>. <strong>Letters in Applied Microbiology, 2021</strong>.</li>
          <li>M. Silva*, <strong>D. Pratas</strong>*, A. J. Pinho. <a href="https://doi.org/10.3390/e23050530">AC2: An Efficient Protein Sequence Compression Tool Using Artificial Neural Networks and Cache-Hash Models</a>. <strong>Entropy, 2021</strong>.</li>
          <li>J. M. Silva, <strong>D. Pratas</strong>, R. Antunes, S. Matos, A. J. Pinho. <a href="https://doi.org/10.1016/j.patcog.2021.107864">Automatic analysis of artistic paintings using information-based measures</a>. <strong>Pattern Recognition, 2021</strong>.</li>
          <li>J. R. Almeida, <strong>D. Pratas</strong>, J. L. Oliveira. <a href="https://doi.org/10.1016/j.compbiomed.2020.104180">A semi-automatic methodology for analysing distributed and private biobanks</a>. <strong>Computers in Biology and Medicine, 2021</strong>.</li>
          <li><strong>D. Pratas</strong>*, J. M. Silva. <a href="https://doi.org/10.1093/bioinformatics/btaa686">Persistent minimal sequences of SARS-CoV-2</a>. <strong>Bioinformatics, 2020</strong>.</li>
          <li><strong>D. Pratas</strong>*, M. Toppinen, L. Pyöriä, K. Hedman, A. Sajantila, M. F. Perdomo. <a href="https://doi.org/10.1093/gigascience/giaa086">A hybrid pipeline for reconstruction and analysis of viral genomes at multi-organ level</a>. <strong>GigaScience, 2020</strong>.</li>
          <li>M. Silva*, <strong>D. Pratas</strong>*, A. J. Pinho. <a href="https://doi.org/10.1093/gigascience/giaa119">Efficient DNA sequence compression with neural networks</a>. <strong>GigaScience, 2020</strong>.</li>
          <li>M. Toppinen, <strong>D. Pratas</strong>, E. Väisänen, M. Söderlund-Venermo, K. Hedman, M. F. Perdomo, A. Sajantila. <a href="https://doi.org/10.1016/j.fsigen.2020.102353">The landscape of persistent human DNA viruses in femoral bone</a>. <strong>Forensic Science International: Genetics, 2020</strong>.</li>
          <li>M. Hosseini, <strong>D. Pratas</strong>, B. Morgenstern, A. J. Pinho. <a href="https://doi.org/10.1093/gigascience/giaa048">Smash++: an alignment-free and memory-efficient tool to find genomic rearrangements</a>. <strong>GigaScience, 2020</strong>.</li>
          <li>J. R. Almeida, A. J. Pinho, J. L. Oliveira, O. Fajarda, <strong>D. Pratas</strong>. <a href="https://doi.org/10.1016/j.softx.2020.100535">GTO: A toolkit to unify pipelines in genomic and proteomic research</a>. <strong>SoftwareX, 2020</strong>.</li>
          <li>J. M. Silva, E. Pinho, S. Matos, <strong>D. Pratas</strong>. <a href="https://doi.org/10.3390/e22010105">Statistical Complexity Analysis of Turing Machine Tapes with Fixed Algorithmic Complexity Using the Best-Order Markov Model</a>. <strong>Entropy, 2020</strong>.</li>
          <li><strong>D. Pratas</strong>*, M. Hosseini, J. M. Silva, A. J. Pinho. <a href="https://doi.org/10.3390/e21111074">A reference-free lossless compression algorithm for DNA sequences using a competitive prediction of two classes of weighted models</a>. <strong>Entropy, 2019</strong>.</li>
          <li>M. Hosseini, <strong>D. Pratas</strong>, A. J. Pinho. <a href="https://doi.org/10.1093/bioinformatics/bty645">Cryfa: a secure encryption tool for genomic data</a>. <strong>Bioinformatics, 2019</strong>.</li>
          <li>M. Hosseini, <strong>D. Pratas</strong>, A. J. Pinho. <a href="https://doi.org/10.1007/s12539-019-00322-1">AC: A Compression Tool for Amino Acid Sequences</a>. <strong>Interdisciplinary Sciences: Computational Life Sciences, 2019</strong>.</li>
          <li>J. M. Carvalho, S. Brás, <strong>D. Pratas</strong>, J. Ferreira, S. C. Soares, A. J. Pinho. <a href="https://doi.org/10.1016/j.patrec.2018.05.026">Extended-Alphabet Finite-Context Models</a>. <strong>Pattern Recognition Letters, 2018</strong>.</li>
          <li><strong>D. Pratas</strong>*, M. Hosseini, G. Grilo, A. J. Pinho, R. Silva, T. Caetano, J. Carneiro, F. Pereira. <a href="https://doi.org/10.3390/genes9090445">Metagenomic Composition Analysis of an Ancient Sequenced Polar Bear Jawbone from Svalbard</a>. <strong>Genes, 2018</strong>.</li>
          <li><strong>D. Pratas</strong>*, R. Silva, A. J. Pinho. <a href="https://doi.org/10.3390/e20060393">Comparison of Compression-Based Measures with Application to the Evolution of Primate Genomes</a>. <strong>Entropy, 2018</strong>.</li>
          <li>M. Hosseini, <strong>D. Pratas</strong>, A. J. Pinho. <a href="https://doi.org/10.3390/info7040056">A Survey on Data Compression Methods for Biological Sequences</a>. <strong>Information, 2016</strong>.</li>
          <li><strong>D. Pratas</strong>*, R. Silva, A. J. Pinho, P. J. S. G. Ferreira. <a href="https://doi.org/10.1038/srep10203">An alignment-free method to find and visualise rearrangements between pairs of DNA sequences</a>. <strong>Scientific Reports, 2015</strong>.</li>
          <li>R. Silva*, <strong>D. Pratas</strong>*, L. Castro, A. J. Pinho, P. J. S. G. Ferreira. <a href="https://doi.org/10.1093/bioinformatics/btv189">Three minimal sequences found in Ebola virus genomes and absent from human DNA</a>. <strong>Bioinformatics, 2015</strong>.</li>
          <li>L. Matos, A. J. R. Neves, <strong>D. Pratas</strong>, A. J. Pinho. <a href="https://doi.org/10.1371/journal.pone.0116082">MAFCO: A compression tool for MAF files</a>. <strong>PLoS ONE, 2015</strong>.</li>
          <li>A. J. Pinho, <strong>D. Pratas</strong>. <a href="https://doi.org/10.1093/bioinformatics/btt594">MFCompress: a compression tool for FASTA and multi-FASTA data</a>. <strong>Bioinformatics, 2014</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho, J. M. O. S. Rodrigues. <a href="https://doi.org/10.1186/1756-0500-7-40">XS: a FASTQ read simulator</a>. <strong>BMC Research Notes, 2014</strong>.</li>
          <li>A. J. Pinho, S. P. Garcia, <strong>D. Pratas</strong>, P. J. S. G. Ferreira. <a href="https://doi.org/10.1371/journal.pone.0079922">DNA sequences at a glance</a>. <strong>PLoS ONE, 2013</strong>.</li>
          <li>L. Matos, <strong>D. Pratas</strong>, A. J. Pinho. <a href="https://doi.org/10.1109/TIT.2012.2236605">A compression model for DNA multiple sequence alignment blocks</a>. <strong>IEEE Transactions on Information Theory, 2013</strong>.</li>
          <li>S. P. Garcia, J. M. O. S. Rodrigues, S. Santos, <strong>D. Pratas</strong>, V. Afreixo, C. A. C. Bastos, P. J. S. G. Ferreira, A. J. Pinho. <a href="https://doi.org/10.1109/tcbb.2013.77">A genomic distance for assembly comparison based on compressed maximal exact matches</a>. <strong>IEEE Transactions on Computational Biology and Bioinformatics, 2013</strong>.</li>
          <li>A. J. Pinho, <strong>D. Pratas</strong>, S. P. Garcia. <a href="https://doi.org/10.1093/nar/gkr1124">GReEn: a tool for efficient compression of genome resequencing data</a>. <strong>Nucleic Acids Research, 2012</strong>.</li>
        </ol>

        <h3>International Conference Articles:</h3>
        <ol>
          <li>D. Yamunaque, A. J. Pinho, A. Sajantila, <strong>D. Pratas</strong>. <a href="https://doi.org/10.1007/978-3-031-99568-2_5">A Machine Learning Method for Authentication of Human Ancient Mitochondrial DNA</a>. IbPRIA 2025, Coimbra, Portugal, July <strong>2025</strong>.</li>
          <li>D. Lei, D. Yamunaque, A. J. Pinho, <strong>D. Pratas</strong>. <a href="https://doi.org/10.1007/978-3-031-99568-2_24">ECOmpress: A web tool for boosting energy efficiency through data compression</a>. IbPRIA 2025, Coimbra, Portugal, July <strong>2025</strong>.</li>
          <li>A. J. Pinho, <strong>D. Pratas</strong>. <a href="https://doi.org/10.1109/DCC62719.2025.00082">Optimization of data compression parameters using genetic algorithms</a>. DCC 2025, Snowbird, United States, March <strong>2025</strong>.</li>
          <li>L. Almeida, P. Rodrigues, D. Magalhães, A. J. Pinho, <strong>D. Pratas</strong>. <a href="https://doi.org/10.1109/DCC62719.2025.00046">AIDetx: a compression-based method for identification of machine-learning generated text</a>. DCC 2025, Snowbird, United States, March <strong>2025</strong>.</li>
          <li>M. J. P. Sousa, A. J. Pinho, <strong>D. Pratas</strong>. <a href="https://doi.org/10.23919/EUSIPCO63174.2024.10715352">Improving the generation of viral consensus sequences using adaptive models</a>. EUSIPCO 2024, Lyon, France, August <strong>2024</strong>.</li>
          <li>M. J. P. Sousa, A. J. Pinho, <strong>D. Pratas</strong>*. <a href="https://doi.org/10.23919/EUSIPCO63174.2024.10715165">A sensitive compression-based method for filtering targeted FASTQ sequencing reads</a>. EUSIPCO 2024, Lyon, France, August <strong>2024</strong>.</li>
          <li>T. Fonseca, M. J. P. Sousa, A. J. Pinho, <strong>D. Pratas</strong>. <a href="https://doi.org/10.23919/EUSIPCO63174.2024.10715012">A sorting tool for improving FASTA data compression tools</a>. EUSIPCO 2024, Lyon, France, August <strong>2024</strong>.</li>
          <li>A. J. Pinho, <strong>D. Pratas</strong>. <a href="https://doi.org/10.1109/DCC58796.2024.00042">Copy models for protein sequence compression</a>. DCC 2024, Snowbird, United States, March <strong>2024</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho. <a href="https://doi.org/10.1109/DCC58796.2024.00095">An experimental sorting method for improving metagenomic data encoding</a>. DCC 2024, Snowbird, United States, March <strong>2024</strong>.</li>
          <li>M. J. P. Sousa, <strong>D. Pratas</strong>. <a href="https://doi.org/10.23919/EUSIPCO63174.2024.10715352">A method for improving the generation of consensus sequences</a>. Workshop on Informatics Engineering Research, Porto, Portugal, <strong>2024</strong>.</li>
          <li>J. M. Silva, <strong>D. Pratas</strong>, S. Matos. <a href="https://doi.org/10.1007/978-3-031-36007-7_12">Exploring Kolmogorov Complexity Approximations for Data Analysis: Insights and Applications</a>. DoCEIS 2023, Caparica, Portugal, pp. 161–174, <strong>2023</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho. <a href="https://doi.org/10.1109/DCC55655.2023.00037">JARVIS2: a data compressor for large genome sequences</a>. DCC 2023, Snowbird, United States, March <strong>2023</strong>.</li>
          <li>J. M. Silva, <strong>D. Pratas</strong>, T. Caetano, S. Matos. <a href="https://doi.org/10.1007/978-3-031-04881-4_25">Feature-Based Classification of Archaeal Sequences Using Compression-Based Methods</a>. IbPRIA 2022, Aveiro, Portugal, May <strong>2022</strong>.</li>
          <li>M. Hosseini, <strong>D. Pratas</strong>, A. J. Pinho. <a href="https://doi.org/10.23919/EUSIPCO.2019.8902695">A probabilistic method to find and visualize distinct regions in protein sequences</a>. EUSIPCO 2019, A Coruña, Spain, September <strong>2019</strong>.</li>
          <li><strong>D. Pratas</strong>*, M. Hosseini, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-030-23873-5_17">GeCo2: An optimized tool for lossless compression and analysis of DNA sequences</a>. PACBB 2019, Ávila, Spain, June <strong>2019</strong>.</li>
          <li><strong>D. Pratas</strong>*, M. Hosseini, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-030-23873-5_16">Visualization of similar primer and adapter sequences in assembled archaeal genomes</a>. PACBB 2019, Ávila, Spain, June <strong>2019</strong>.</li>
          <li>A. J. Pinho, <strong>D. Pratas</strong>. <a href="https://doi.org/10.1007/978-3-030-01449-0_41">An application of data compression models to handwritten digit classification</a>. ACIVS 2018, September <strong>2018</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho. <a href="https://doi.org/10.23919/EUSIPCO.2018.8553297">Metagenomic composition analysis of sedimentary ancient DNA from the Isle of Wight</a>. EUSIPCO 2018, Rome, Italy, September <strong>2018</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-319-98702-6_25">A DNA sequence corpus for compression benchmark</a>. PACBB 2018, Toledo, Spain, June <strong>2018</strong>.</li>
          <li><strong>D. Pratas</strong>*, M. Hosseini, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-319-98702-6_13">Compression of amino acid sequences</a>. PACBB 2018, Toledo, Spain, June <strong>2018</strong>.</li>
          <li>M. Gaspar, <strong>D. Pratas</strong>, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-319-98702-6_14">NET-ASAR: a tool for DNA sequence search based on data compression</a>. PACBB 2018, Toledo, Spain, June <strong>2018</strong>.</li>
          <li><strong>D. Pratas</strong>*, M. Hosseini, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-319-60816-7_37">Cryfa: a tool to compact and encrypt FASTA files</a>. PACBB 2017, Porto, Portugal, June <strong>2017</strong>.</li>
          <li>M. Hosseini, <strong>D. Pratas</strong>, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-319-60816-7_28">On the role of inverted repeats in DNA sequence similarity</a>. PACBB 2017, Porto, Portugal, June <strong>2017</strong>.</li>
          <li><strong>D. Pratas</strong>*, M. Hosseini, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-319-60816-7_32">Substitutional tolerant Markov models for relative compression of DNA sequences</a>. PACBB 2017, Porto, Portugal, June <strong>2017</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-319-58838-4_29">On the approximation of the Kolmogorov complexity for DNA sequences</a>. IbPRIA 2017, Faro, Portugal, June <strong>2017</strong>.</li>
          <li><strong>D. Pratas</strong>*, M. Hosseini, R. Silva, A. J. Pinho, P. J. S. G. Ferreira. <a href="https://doi.org/10.1007/978-3-319-58838-4_26">Visualization of distinct DNA regions of the modern human relative to a Neanderthal genome</a>. IbPRIA 2017, Faro, Portugal, June <strong>2017</strong>.</li>
          <li>A. J. Pinho, <strong>D. Pratas</strong>, P. J. S. G. Ferreira. <a href="https://doi.org/10.1109/DCC.2016.53">Authorship attribution using relative compression</a>. DCC 2016, Snowbird, United States, March <strong>2016</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho, P. J. S. G. Ferreira. <a href="https://doi.org/10.1109/DCC.2016.60">Efficient compression of genomic sequences</a>. DCC 2016, Snowbird, United States, March <strong>2016</strong>.</li>
          <li>A. J. Pinho, <strong>D. Pratas</strong>, P. J. S. G. Ferreira. <a href="https://doi.org/10.1007/978-3-319-11758-4_4">A new compressor for measuring distances among images</a>. ICIAR 2014, Vilamoura, Portugal, October <strong>2014</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho. <a href="https://ieeexplore.ieee.org/document/6952879">Exploring deep Markov models in genomic data compression using sequence pre-analysis</a>. EUSIPCO 2014, Lisbon, Portugal, September <strong>2014</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho. <a href="https://doi.org/10.1109/DCC.2014.58">A conditional compression distance that unveils insights of the genomic evolution</a>. DCC 2014, Snowbird, United States, March <strong>2014</strong>.</li>
          <li>A. J. Pinho, <strong>D. Pratas</strong>, P. J. S. G. Ferreira. <a href="https://doi.org/10.1109/DCC.2014.54">Information profiles for DNA pattern discovery</a>. DCC 2014, Snowbird, United States, March <strong>2014</strong>.</li>
          <li>S. P. Garcia, J. M. O. S. Rodrigues, <strong>D. Pratas</strong>, A. J. Pinho. <a href="">Comparing maximal exact repeats in human genome assemblies using a normalized compression distance</a>. ISMB 2012, Long Beach, United States, July <strong>2012</strong>.</li>
          <li>L. Matos, <strong>D. Pratas</strong>, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-642-31295-3_42">Compression of whole genome alignments using a mixture of finite-context models</a>. ICIAR 2012, Aveiro, Portugal, June <strong>2012</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-642-31295-3_19">On the Detection of Unknown Locally Repeating Patterns in Images</a>. ICIAR 2012, Aveiro, Portugal, June <strong>2012</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho, S. P. Garcia. <a href="https://doi.org/10.1007/978-3-642-28839-5_25">Exon: A web-based software toolkit for DNA sequence analysis</a>. PACBB 2012, Salamanca, Spain, March <strong>2012</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho, S. P. Garcia. <a href="https://www.scitepress.org/papers/2012/37802/37802.pdf">Computation of the normalized compression distance of DNA sequences using a mixture of finite-context models</a>. Bioinformatics 2012, Vilamoura, Portugal, February <strong>2012</strong>.</li>
          <li>A. J. Pinho, <strong>D. Pratas</strong>, S. P. Garcia. <a href="https://doi.org/10.1007/978-3-642-25364-5_8">Complexity profiles of DNA sequences using finite-context models</a>. USAB 2011, Graz, Austria, November <strong>2011</strong>.</li>
          <li>A. J. Pinho, <strong>D. Pratas</strong>, P. J. S. G. Ferreira, S. P. Garcia. <a href="https://ieeexplore.ieee.org/document/7074117">Symbolic to numerical conversion of DNA sequences using finite-context models</a>. EUSIPCO 2011, Barcelona, Spain, August <strong>2011</strong>.</li>
          <li>A. J. Pinho, <strong>D. Pratas</strong>, P. J. S. G. Ferreira. <a href="https://doi.org/10.1109/SSP.2011.5967637">Bacteria DNA sequence compression using a mixture of finite-context models</a>. SSP 2011, Nice, France, June <strong>2011</strong>.</li>
          <li><strong>D. Pratas</strong>*, C. A. C. Bastos, A. J. Pinho, A. J. R. Neves, L. Matos. <a href="https://doi.org/10.1109/SSP.2011.5967639">DNA synthetic sequences generation using multiple competing Markov models</a>. SSP 2011, Nice, France, June <strong>2011</strong>.</li>
          <li><strong>D. Pratas</strong>*, A. J. Pinho. <a href="https://doi.org/10.1007/978-3-642-19914-1_29">Compressing the human genome using exclusively Markov models</a>. PACBB 2011, Salamanca, Spain, April <strong>2011</strong>.</li>
        </ol>

        <h3>National Conference Articles</h3>
        <ol>
          <li>A. Martins, <strong>D. Pratas</strong>, A. Pinho, S. Gouveia. <a href="https://aprp.pt/wp-content/uploads/2025/01/RECPAD24_Proceedings.pdf">Finding relevant features to enhance compression of categorical time series</a>. RecPad 2024, Covilhã, Portugal, October <strong>2024</strong>.</li>
          <li>M. J. P. Sousa, <strong>D. Pratas</strong>. <a href="https://aprp.pt/wp-content/uploads/2023/10/ProceedingsRecPad2023.pdf">A method for accurate reconstruction of persistent human viral sequences</a>. RecPad 2023, Coimbra, Portugal, October <strong>2023</strong>.</li>
          <li>M. J. P. Sousa, <strong>D. Pratas</strong>. <a href="https://aprp.pt/wp-content/uploads/2022/11/ProceedingsRecpad2022_v3.pdf">A survey on computational tools for human viral genomes reconstruction</a>. RecPad 2022, Leiria, Portugal, October <strong>2022</strong>.</li>
          <li>M. J. P. Sousa, R. Ferrolho, T. Fonseca, A. J. Pinho, <strong>D. Pratas</strong>. <a href="https://aprp.pt/wp-content/uploads/2022/11/ProceedingsRecpad2022_v3.pdf">Improving the compression of a complete Telomere-to-Telomere (T2T) human genome sequence</a>. RecPad 2022, Leiria, Portugal, October <strong>2022</strong>.</li>
          <li>J. M. Silva, <strong>D. Pratas</strong>, T. Caetano, S. Matos. <a href="https://aprp.pt/wp-content/uploads/proceedings_recpad2021.pdf">Archaea Taxonomic Classification</a>. RecPad 2021, Évora, Portugal, November <strong>2021</strong>.</li>
          <li>J. M. Silva, <strong>D. Pratas</strong>, S. Matos. <a href="https://aprp.pt/wp-content/uploads/proceedings_recpad2020.pdf">Comparison and Evaluation of Information-based Measures in Images</a>. RecPad 2020, Évora, Portugal, October <strong>2020</strong>.</li>
          <li>M. Hosseini, <strong>D. Pratas</strong>, A. J. Pinho. <a href="https://aprp.pt/wp-content/uploads/proceedings_recpad2019.pdf">Clustering DNA sequences by relative compression</a>. RecPad 2019, Porto, Portugal, October <strong>2019</strong>.</li>
          <li>J. M. Silva, <strong>D. Pratas</strong>, S. Matos. <a href="https://aprp.pt/wp-content/uploads/proceedings_recpad2019.pdf">Evaluation of Statistical Complexity in Viral Genome Sequences</a>. RecPad 2019, Porto, Portugal, October <strong>2019</strong>.</li>
          <li>M. Hosseini, <strong>D. Pratas</strong>, A. Amorim, J. Carneiro. <a href="https://www.researchgate.net/publication/337592393_Improving_the_detection_of_mtDNA_rearrangements_using_a_fast_and_accurate_algorithm">Improving the detection of mtDNA rearrangements using a fast and accurate algorithm</a>. ENBE 2019, Porto, Portugal, November <strong>2019</strong>.</li>
          <li>A. Teixeira, <strong>D. Pratas</strong>, A. J. Pinho, R. Silva. <a href="https://aprp.pt/wp-content/uploads/proceedings_recpad2018.pdf">Evolutionary insights from the comparative analysis of hominid genomes</a>. RecPad 2018, Coimbra, October <strong>2018</strong>.</li>
          <li>C. Figueiredo, <strong>D. Pratas</strong>, A. J. Pinho, R. Silva. <a href="https://aprp.pt/wp-content/uploads/proceedings_recpad2018.pdf">Identification of antifungal targets using alignment-free methods</a>. RecPad 2018, Coimbra, October <strong>2018</strong>.</li>
          <li><strong>D. Pratas*</strong>, R. Silva, A. J. Pinho, P. J. S. G. Ferreira. <a href="https://recpad2015.ualg.pt/Procs/RecPad2015.pdf">Detection and visualization of regions of human DNA not present in other primates</a>. RecPad 2015, Faro, Portugal, October <strong>2015</strong>.</li>
          <li><strong>D. Pratas*</strong>, R. Silva, A. J. Pinho. <a href="https://aprp.pt/wp-content/uploads/recpad2014.pdf">Large-scale inversions between human reference assemblies</a>. RecPad 2014, Covilhã, Portugal, October <strong>2014</strong>.</li>
          <li>R. Silva, L. Castro, <strong>D. Pratas</strong>, A. J. Pinho. <a href="https://aprp.pt/wp-content/uploads/recpad2014.pdf">Towards personalized medicine: ebola virus absent words in the human genome</a>. RecPad 2014, Covilhã, Portugal, October <strong>2014</strong>.</li>
          <li><strong>D. Pratas*</strong>, A. J. Pinho. <a href="https://aprp.pt/wp-content/uploads/Documents/RecPad2013_proceedings.pdf">Insights into primates genomic evolution using a compression distance</a>. RecPad 2013, Lisbon, November <strong>2013</strong>.</li>
          <li><strong>D. Pratas*</strong>, A. J. Pinho. <a href="https://sweet.ua.pt/pratas/papers/Pratas-2012c.pdf">On the compression of FASTQ quality-scores</a>. RecPad 2012, Coimbra, Portugal, October <strong>2012</strong>.</li>
          <li><strong>D. Pratas*</strong>, A. J. Pinho. <a href="https://paginas.fe.up.pt/~prodei/dsie12/papers/paper_6.pdf">M6: a method for compressing complete genomes using Markov models</a>. DSIE 2012, Porto, Portugal, January <strong>2012</strong>.</li>
          <li><strong>D. Pratas*</strong>, S. P. Garcia, A. J. Pinho. <a href="https://sweet.ua.pt/pratas/papers/Pratas-2011c.pdf">Analysis of patterns in S. pombe genome through compression-based complexity profiles</a>. RecPad 2011, Porto, Portugal, October <strong>2011</strong>.</li>
          <li><strong>D. Pratas*</strong>, A. J. Pinho. <a href="https://sweet.ua.pt/pratas/papers/Pratas-2010b.pdf">Analysis of DNA sequences using finite-context modelling and compression</a>. RecPad 2010, Vila Real, Portugal, October <strong>2010</strong>.</li>
          <li><strong>D. Pratas*</strong>, A. J. Pinho, A. J. R. Neves, C. A. C. Bastos. <a href="https://sweet.ua.pt/pratas/papers/Pratas-2010a.pdf">DNA synthetic sequences generated by finite-context models</a>. RecPad 2010, Vila Real, Portugal, October <strong>2010</strong>.</li>
        </ol>

        <h3>Book Chapters</h3>
        <ol>
          <li>A. J. Pinho, <strong>D. Pratas</strong>, S. P. Garcia. <a href="https://doi.org/10.1007/978-1-62703-514-9_2">Compressing resequencing data with GReEn</a>. In: Deep Sequencing Data Analysis, ed. Noam Shomron, Humana Press (Methods in Molecular Biology, Vol. 1038), pp. 27–37, July <strong>2013</strong>.</li>
        </ol>

        <h3>Other</h3>
        <ol>
          <li>M. J. P. Sousa, M. Toppinen, L. Pyöriä, K. Hedman, A. Sajantila, M. F. Perdomo, <strong>D. Pratas*</strong>. <a href="https://doi.org/10.1101/2025.01.17.633368">Comparative evaluation of computational methods for reconstruction of human viral genomes</a>. bioRxiv, <strong>2025</strong>.</li>
          <li><strong>D. Pratas*</strong>, A. J. Pinho, R. M. Silva, J. M. O. S. Rodrigues, M. Hosseini, T. Caetano, P. J. S. G. Ferreira. <a href="https://doi.org/10.1101/267179">FALCON-meta: A method to infer metagenomic composition of ancient DNA</a>. bioRxiv, <strong>2018</strong>.</li>
        </ol>

        <h3>Books</h3>
        <ol>
          <li>D. Figueiredo, C. Martín-Vide, <strong>D. Pratas</strong>, M. A. Vega-Rodríguez. <a href="https://doi.org/10.1007/978-3-319-58163-7">Algorithms for Computational Biology</a>. Springer International Publishing, June <strong>2017</strong>.</li>
          <li><strong>D. Pratas</strong>. Compression and analysis of genomic data. PhD Thesis, University of Aveiro, Portugal, <strong>2016</strong>.</li>
        </ol>
    </section>

    <section id="Software">
      <h2>Software</h2>
      <div class="software-grid">
        <div class="software-card">
          <a href="https://github.com/viromelab/tracespipe"><img src="imgs/tracespipe.png" alt="TRACESPipe" /></a>
          <div class="content">
            <p align="justify"><b>TRACESPipe</b> is a hybrid pipeline for efficient reconstruction and analysis of viral and host genomes that can be set at multi-organ level for clinical or aDNA purposes.</p>
          </div>
        </div>
        <div class="software-card">
          <a href="https://github.com/cobilab/altair"><img src="imgs/altair.png" alt="AltaiR" /></a>
          <div class="content">
            <p align="justify"><b>AltaiR</b> is a fast-flexible C toolkit for alignment-free temporal analysis of multi-FASTA data in large genomic collections and for use in potential epidemic scenarios.</p>
          </div>
        </div>
        <div class="software-card">
          <a href="https://github.com/cobilab/alcor"><img src="imgs/alcor.png" alt="AlcoR" /></a>
          <div class="content">
            <p align="justify"><b>AlcoR</b> is an alignment-free toolkit for analysis of low-complexity regions in biological data, supporting mapping, masking, simulation and visualization.</p>
          </div>
        </div>
        <div class="software-card">
          <a href="https://github.com/cobilab/jarvis3"><img src="imgs/jarvis3.png" alt="JARVIS3" /></a>
          <div class="content">
            <p align="justify"><b>JARVIS3</b> is an efficient lossless data (de)compression tool tailored for genomic sequences with extension for compressing FASTA and FASTQ data.</p>
          </div>
        </div>
        <div class="software-card">
          <a href="https://github.com/cobilab/falcon"><img src="imgs/falcon.png" alt="FALCON" /></a>
          <div class="content">
            <p align="justify"><b>FALCON</b> is an ultra-fast method to infer metagenomic composition of sequenced reads while minimizing false positives and maximizing accuracy.</p>
          </div>
        </div>
        <div class="software-card">
          <a href="https://github.com/cobilab/spark"><img src="imgs/spark.png" alt="SPARK" /></a>
          <div class="content">
            <p align="justify"><b>SPARK</b> is a toolkit to simulate, search, and analyze exact or approximate Turing Machines using alignment-free methods and colorized visualizations.</p>
          </div>
        </div>
        <div class="software-card">
          <a href="https://github.com/cobilab/cryfa"><img src="imgs/cryfa.png" alt="CRYFA" /></a>
          <div class="content">
            <p align="justify"><b>CRYFA</b> is an ultrafast encryption tool specifically designed for genomic data, while it can also compress FASTA / FASTQ data by a factor of three.</p>
          </div>
        </div>
        <div class="software-card">
          <a href="https://github.com/cobilab/smash"><img src="imgs/smash.png" alt="CRYFA" /></a>
          <div class="content">
            <p align="justify"><b>SMASH</b> is a compression-based and alignment-free method to automatically find and visualise rearrangements between pairs of DNA sequences.</p>
          </div>
        </div>
        <div class="software-card">
          <a href="https://github.com/cobilab/ac2"><img src="imgs/ac2.png" alt="CRYFA" /></a>
          <div class="content">
            <p align="justify"><b>AC</b> is a lossless compressor to compress efficiently amino acid sequences (proteins). It uses a cooperation between multiple context models.</p>
          </div>
        </div>      
        <div class="software-card">
          <a href="https://github.com/cobilab/geco3"><img src="imgs/geco3.png" alt="GeCo3" /></a>
          <div class="content">
            <p align="justify"><b>GeCo3</b> is an efficient lossless genomic compressor that uses a neural network for expert mixing. It supports relative and conditional compression.</p>
          </div>
        </div>    
        <div class="software-card">
          <a href="https://github.com/cobilab/chester"><img src="imgs/chester.png" alt="CHESTER" /></a>
          <div class="content">
            <p align="justify"><b>Chester</b> is a probabilistic tool that uses Bloom filters to map and further compute visualization of evolutionary regions (relative singularity regions).</p>
          </div>
        </div>    
        <div>
          <p align="center"><a href="https://github.com/cobilab/"><font size="6">[+]</font><br>More software is available at Github</a>.</p>
        </div>
      </div>
    </section>

    <section id="Datasets">
      <h2>Datasets</h2>
      <div class="software-grid">
        <div class="software-card">
          <a href="http://sweet.ua.pt/pratas/datasets/DNACorpus.zip"><img src="imgs/dna.jpg" alt="DNA Corpus" /></a>
          <div class="content">
            <p align="center"><b>DNA sequence corpus in sequence format (ACGT)</b></p>
          </div>
        </div>
        <div class="software-card">
          <a href="http://sweet.ua.pt/pratas/datasets/AminoAcidsCorpus.zip"><img src="imgs/aa.jpg" alt="AA Corpus" /></a>
          <div class="content">
            <p align="center"><b>Amino Acids sequence corpus (sequence only)</b></p>
          </div>
        </div>
        <div class="software-card">
          <a href="https://github.com/viromelab/TRACESPipeLite/raw/refs/heads/main/src/VDB.mfa.lzma"><img src="imgs/vdb.jpg" alt="Human Virus Genome Database" /></a>
          <div class="content">
            <p align="center"><b>Human virus genome sequences (FASTA format)</b></p>
          </div>
        </div>
        <div class="software-card">
          <a href="data/CoinsData.zip"><img src="imgs/coins.png" alt="250 Portuguese coins dataset" /></a>
          <div class="content">
            <p align="center"><b>XRF, UV-VIS, and Weight Data for 250 Portuguese Coins</b></p>
          </div>
        </div>
      </div>
    </section>

    <section id="Classes">
      <h2>Classes</h2>

      <h3>Algorithmic Information Theory</h3>
      <article class="ait-container">
        <p align="justify">
        <img class="ait-container__img" src="imgs/ait.png" alt="Algorithmic Information Theory illustration" width="320" height="240" />
        Algorithmic Information Theory  (AIT, or TAI in Portuguese) is a field at the intersection of computer science, mathematics, and information theory that explores how information can be measured, represented, and processed using algorithms. It provides a deep understanding of concepts such as data compression, randomness, machine learning, and the limits of computation. AIT equips students with fundamental tools to reason about what information is, how to model sources of data, and how efficiently data can be encoded or processed. These ideas are essential for anyone working in areas like machine learning, cryptography, data science, and theoretical computer science.
        The AIT course at the University of Aveiro combines both theoretical and practical components. In addition to lectures, students engage in hands-on learning through three group-based practical assignments, where they apply the concepts studied in class to real problems. This structure helps students to consolidate their understanding and develop collaborative problem-solving skills that are valuable in both research and industry.
        </p>
      </article>
    </section>

    <section id="Challenge">
      <h2>Challenge</h2>

      <h3>Human genome sequence compression</h3>
      <article class="cha-container">
        <p align="justify">
        <img class="cha-container__img" src="imgs/hgsc.png" alt="Human Genome Sequence Challenge" width="320" height="240" />
        The challenge is to develop a data compressor that improves the lossless, reference-free minimal representation of a complete human genome sequence, specifically version 2.0 of the <a href="https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/analysis_set/chm13v2.0.fa.gz">T2T Chm13 human genome</a>. 
        This fully sequenced genome is described in detail in the corresponding <a href="https://doi.org/10.1126/science.abj6987">scientific article</a>, which serves as the foundation for the task. Participants are invited to propose innovative compression methods that advance the state of the art, with results compared against existing solutions on the <a href="https://github.com/cobilab/HumanGenome">current leaderboard</a>. Each of the compressor and decompressor must complete within 48 hours (wall-clock) and use no more than 32 GB of RAM (peak). The leading entries are currently dominated by the JARVIS series—specifically JARVIS3 and JARVIS2. These DNA-based compressors are capable of reducing the size of the human genome by nearly one-third. JARVIS3 achieves approximately 7% greater compression compared to <em>nncp</em>, a Transformer-based general-purpose data compression tool.
        </p>
      </article>
    </section> 

    <section id="Contact">
      <h2>Contact</h2>
  
        <h3>Address</h3>  
          <p align="left">  
          IEETA, University of Aveiro,<br>
          Campus Universitário Santiago, 3810-193<br>
          Aveiro, Portugal.
          </p>
        <h3>Email</h3>    
          <p><a href="mailto:pratas@ua.pt">pratas@ua.pt</a></p>
        <h3>Phone</h3>  
          <p>+351 234 370 506</p>
        <h3>Affiliation</h3>
          <p align="left">
          IEETA/LASI, Institute of Electronics and Informatics Engineering of Aveiro, University of Aveiro.<br>
          DETI, Department of Electronics, Telecommunications and Informatics, University of Aveiro.<br>
          DV, Department of Virology, University of Helsinki.
          </p>
    </section>
  </main>

  <footer>
    <p>&copy; 2025 D. Pratas, University of Aveiro &#38; University of Helsinki</p>
    <button id="topBtn" title="Go to Top">↑Top</button>
  </footer>
  
  <script src="js/scripts.js"></script>
</body>
</html>
