<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>EAGLE: an alignment-free tool to find relative absent words</title>
    <link rel="icon" href="https://raw.githubusercontent.com/pratas/eagle/master/logoTrans.png"> 
    <link rel="stylesheet" href="../stylesheets/stylesRed.css">
    <link rel="stylesheet" href="../stylesheets/pygment_trac.css">
    <script src="../javascripts/scale.fix.js"></script>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <a href="https://github.com/pratas/eagle"><img src="https://raw.githubusercontent.com/pratas/eagle/master/logoTrans.png" alt="EAGLE" width="150"></a>
        <p class="header">An alignment-free tool to find relative absent words (RAWs) in genomic sequences</p>
        <ul>
          <li class="download"><a class="buttons" href="https://github.com/pratas/eagle/zipball/master">Download ZIP</a></li>
          <li class="download"><a class="buttons" href="https://github.com/pratas/eagle/tarball/master">Download TAR</a></li>
          <li><a class="buttons github" href="http://github.com/pratas/eagle">View On GitHub</a></li>
        </ul>
      </header>
      <section>
      <h1>Introduction</h1>
      <h3>What is EAGLE ?</h3>
      <p align="justify"><a href="https://github.com/pratas/eagle">EAGLE</a> is an <b>alignment-free method to find relative absent words (RAWs) in genomic sequences using a reference sequence</b>. EAGLE has been published in <a href="http://bioinformatics.oxfordjournals.org/content/31/15/2421.full.pdf">Bioinformatics in May 2015</a>.</p>
It answers the following questions:
<ul>
<li>Whose words of size-k are present in a sequence but absent from another?</li>
<li>Where do they appear?</li>
</ul>
<h3>How do I get EAGLE ?</h3>
Clone our repository and run CMake command:
<pre>
git clone https://github.com/pratas/eagle.git
cd eagle/
cmake .
make
</pre>
Alternatively to CMake, run:
<pre>
git clone https://github.com/pratas/eagle.git
cd eagle/
cp Makefile.linux Makefile
make
</pre>

<h1>Usage</h1>

To see the possible options type
<pre>
./EAGLE
</pre>
or
<pre>
./EAGLE -h
</pre>
These will print the following options:
<pre>
Usage: EAGLE &#60OPTIONS&#62 ... -r [FILE]  [FILE]:&#60...&#62

  -v                       verbose mode,
  -a                       about EAGLE,
  -t                       use multi-threading,
  -i                       use inversions,
  -min &#60k-mer&#62             k-mer minimum size,
  -max &#60k-mer&#62             k-mer maximum size,

  -r  [rFile]              reference file (db),

  [tFile1]:&#60tFile2&#62:&#60...&#62  target file(s).

EAGLE is a fast method/tool to compute relative MAWs.
The input files should be FASTA (.fa) or SEQ [ACGTN].</p>
</pre>


<h1>License</h1>

GPL v3.

For more information see LICENSE file or visit
<pre>http://www.gnu.org/licenses/gpl-3.0.html</pre>

</section>

<!--      <footer>
        <sub>&copy; 2014&ndash;2016 <a href="http://wiki.ieeta.pt/wiki/index.php/Information_Systems_and_Processing">Lab for Information Systems and Processing</a> at <a href="http://www.ieeta.pt">IEETA</a>, <a href="http://www.ua.pt"> University of Aveiro</a>.-->
<!--<br /> Hosted on <a href="http://pages.github.com/">GitHub Pages</a>. Design based upon Dinky theme.-->
        </sub>
      </footer>
    </div>
    <!--[if !IE]><script>fixScale(document);</script><!--<![endif]-->
  </body>
</html>
